webpack/assets/javascripts/react_app/routes/index.js

```diff
@@ -4,19 +4,21 @@ import { routes } from './routes';
 
 let currentLocation = null;
 
-const AppSwitcher = () => (
+const AppSwitcher = ({ metadata }) => (
```

Comment:
as stated above, metadata is now in the redux state so we should avoid this state (prop drilling)